linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_tcp_invalid_ratelimit/rule.yml

```diff
@@ -0,0 +1,58 @@
+documentation_complete: true
+
+prodtype: wrlinux1019,rhel7,rhel8
+
+title: 'Configure kernel To Rate limit sending of duplicate TCP acknowledgments'
+
+description: |-
+    Make sure that the system is configured to rate limit sending of duplicate TCP acknowledgments.
```

Comment:
```suggestion
    Make sure that the system is configured to limit the maximal rate for sending duplicate acknowledgments in response to incoming TCP packets that are for an existing connection but that are invalid due to any of these reasons:

    (a) out-of-window sequence number, (b) out-of-window acknowledgment number, or (c) PAWS (Protection Against Wrapped Sequence numbers) check failure
```

Reply:
Could the official sysctl language be borrowed here as well?

https://sysctl-explorer.net/net/ipv4/tcp_invalid_ratelimit/